SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -387,6 +387,11 @@ void UGlobalStateManager::RemoveSingletonInstance(const AActor* SingletonActor)
 	SingletonClassPathToActorChannels.Remove(SingletonActor->GetClass()->GetPathName());
 }
 
+void UGlobalStateManager::RemoveAllSingletons()
+{
+	SingletonClassPathToActorChannels.Reset();
```

Comment:
The singletons will be re-added when the new map loads?

Reply:
Yarp. When we express interest in them.